BourbonAndBurial/Controllers/OrderController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BourbonAndBurial.Data;
+using BourbonAndBurial.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BourbonAndBurial.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrderController : ControllerBase
+    {
+        readonly OrderRepository _orderRepository;
+
+        public OrderController()
+        {
+            _orderRepository = new OrderRepository();
+        }
+
+        [HttpPost]
+        public ActionResult AddOrder(CreateOrderRequest createRequest)
+        {
+            var repository = new OrderRepository();
+
+            var newOrder = repository.AddOrder(
+                createRequest.CustomerId,
+                createRequest.PaymentTypeId
+                );
+
+            return Created($"/api/order/{newOrder.OrderId}", newOrder);
+        }
+
+        [HttpGet("orders")]
```

Comment:
Did you mean to make the Post request have ```("orders")``` ? it's kind of redundant since the http request it ```api/order```
Postman with orders = ```api/order/orders```
Postman without orders = ```api/order```
If you wanted to make the post request have a distinct route,  i think ```[HttpPost("neworder")]``` might make more sense, but it's up to you.
Did this make any sense????????